docs/Acceptable-Formulae.md

```diff
@@ -5,6 +5,12 @@ Some formulae should not go in
 additional [Interesting Taps and Forks](Interesting-Taps-and-Forks.md) and anyone can start their
 own!
 
+### Supported platforms in `homebrew/core`
+
+The formula needs to build and run on the latest 3 supported macOS Versions (Intel and ARM), and on 64-bit Intel Linux.
```

Comment:
Could consider linking to install docs so people know what these macOS versions are.
```suggestion
The formula needs to build and run on the latest 3 supported macOS versions (x86_64 and Apple Silicon/ARM), and on x86_64 Linux.
```